tla/ccfraft.tla

```diff
@@ -481,6 +487,19 @@ AppendEntriesBatchsize(i, j) ==
      \* This can be redefined to send bigger batches of entries.
     {nextIndex[i][j]}
 
+
+PlausibleSucessorNode(i) ==
+    \* Find a plausible successor node for i
+    \* Note that while CHOOSE is deterministic, it may not pick the same
```

Comment:
If you wanted to remove to `CHOOSE` to allow any plausible successor then you could have `PlausibleSucessorNode(i)` return the set of plausible successors for i. Then when you where calling `dest |-> PlausibleSucessorNode(i)`, you add a `\E j \in PlausibleSucessorNode(i): ` earlier and then do `dest |->j`. If there are multiple possible successors then the `AdvanceCommitIndex` action can be taken for each

Reply:
> If there are multiple possible successors then the AdvanceCommitIndex action can be taken for each

This had failed to click in my head. Thank you, I have made the change.